packages/marginfi-client-v2/src/models/account/pure.ts

```diff
@@ -942,6 +947,20 @@ class MarginfiAccount {
     return { instructions: [ix], keys: [] };
   }
 
+  async makeAccountAuthorityTransferIx(program: MarginfiProgram, newAccountAuthority: PublicKey) : Promise<InstructionsWrapper> {
+    const accountAuthorityTransferIx = await instructions.makeAccountAuthorityTransferIx(
+      program,
+      {
+        marginfiAccountPk: this.address,
+        marginfiGroupPk: this.group,
+        signerPk: this.authority,
+        newAccountAuthorityPk: newAccountAuthority,
+        feePayerPk: this.address, // ?
```

Comment:
```suggestion
        feePayerPk: this.authority,
```

In the context of the SDK, let's simply set this to that for now. This won't be compatible with squads but we can fix that at that time, as I don't know off-hand the logic needed when this gets used through the MS.

Reply:
But do we want the fee payer to be the same as the signer? I imagine a scenario where we could pay for the account authority transfer on behalf of the user?